kapre/composed.py

```diff
@@ -44,6 +44,7 @@ def get_stft_magnitude_layer(
     input_data_format='default',
     output_data_format='default',
     name='stft_magnitude',
+    use_parallel_stft=False,
```

Comment:
We'd need this in the docstring, too, something like.. 
```
use_parallel_stft (bool): Whether to parallelize stft when running on CPU. If `True`, it uses `kapre.backend.parallel_stft()` which uses multi-processing. If `False`, it uses Tensorflow `tf.signal.stft`, which is significant slower than other STFT implementations such as librosa or scipy on CPU. It does not affect the behavior when running on GPUs. 
```

I think the variable name is good.